### PR TITLE
Adjust yelp API for latitude and longitude

### PR DESCRIPTION
**Pull request by tadakane**
## Description / Changes Made

- Change get_our_own_businesses() route to be based on latitude and longitude in the route. 
- Also, format the returned json from yelp to only contain fields we want.
- These fields include: coordinates, location, name, price, rating, image_url, and description

## Portions of this code that utilized AI generation

None

## How to Test

1. Using Postman, use endpoint api/&lt;latitude>/&lt;longitude>
2. Set Postman to a GET method.
3. Before sending request, w/o middleware, you only need to make sure in the Headers tab on Postman to have a Key "Authorization", then Value as  "Bearer <YELP_API_KEY>". The YELP_API_KEY can be obtained at https://www.yelp.com/developers/v3/manage_app. Login with the Munch Google account. With middleware, you need another Header also "Authorization", then value as "Bearer &lt;user token>. If you are using middleware, make sure the user token authorization is listed first in the headers.
4. Now, send request and you should be getting the JSON with the fields we want from yelp.

## Checklist

- [x] I have added/updated relevant documentation, and I have followed the coding style guidelines.
- [x] I have checked for any potential conflicts with other branches and fixed any merge conflicts.

### CODE REVIEW

File: backend/yelpAPI/fakeDataAPI.py [hunk 1]
```diff
@@ -35,19 +39,62 @@ def get_business():
...
-    # Use query filter to get 50 businesses from the random state provided, can change limit query to X amount
+@yelp_bp.route("/api/<latitude>/<longitude>", methods=["GET"])
+def get_our_own_businesses(latitude, longitude):
+    # Get a random integer for the offset
+    random_int = str(random.randint(0, 500))
+    # Use query filter to get 50 businesses from provided latitude and longitude, can change limit query to X amount
+    # offset is from integer above, to get more random options
     response = requests.get(
-        URL + "businesses/search?location=" + random_state + "&limit=50",
+        URL
```
Comment (noodaj) [inline backend/yelpAPI/fakeDataAPI.py:49]: I think you should have this as a formatted string instead of doing + for every part of the URL.

**Review by noodaj — CHANGES_REQUESTED** [review]
Overall, great job on creating the route for getting the recommended locations based on the user's location. This is great since we are replacing the ML model and we can soon have stuff on the user's home page.

**Review by noodaj — APPROVED** [review]
Great work!

**Review by KeenParchment — APPROVED** [review]
Good job on changing the Yelp API route to Latitude and Longitude, replacing the ML model, and formatting the returned JSON from Yelp to contain only the specified fields. Overall, this looks good to merge.

**Pull request merged**